src/StrStore.sol

```diff
@@ -7,7 +7,17 @@ Vm constant vm = Vm(address(uint160(uint256(keccak256("hevm cheat code")))));
 
 function storeString(address target, uint256 slot, string memory strvalue) {
     bytes memory value = bytes(strvalue);
+    storeBytes(target, slot, value);
+}
+
+function storeBytes(address target, uint256 slot, bytes memory value) {
     uint256 length = value.length;
+
+    if (length == 0) {
+        vm.store(target, bytes32(slot), bytes32(0));
+        return;
+    }
```

Comment:
why is this special case needed?

Reply:
For some reason `storeBytes` was behaving strangely with empty bytes, this addition fixed the problem

Reply:
We could add some unit tests for `storeBytes` in `StrStore.t.sol` to detect what was/is the problem. This feels like a workaround for (maybe) a problem somewhere else. Given the low-level byte manipulation we are doing, we need to ensure this does not hide any problem elsewhere.

Reply:
Hmmm, tbh I don't remember exactly why I added this...

Looking at it right now, it should behave exactly like the `if (length <= 31)` block:
```solidity
        // this would be the same as: slotValue = bytes32(new bytes(0)) | bytes32(0) = bytes32(0)
        bytes32 slotValue = bytes32(value) | bytes32(length * 2);
        // thus, this would actually do: storeBytes32(target, slot, bytes32(0)), which is the same
        storeBytes32(target, slot, slotValue);
        return;
```

Reply:
I removed the `if (length == 0)` block, cause it seemed to make no difference when running the unit tests